lib/temporal/metadata/activity.rb

```diff
@@ -23,6 +23,21 @@ def initialize(namespace:, id:, name:, task_token:, attempt:, workflow_run_id:,
       def activity?
         true
       end
+
+      def to_h
+        {
+          'attempt' => attempt,
+          'headers' => headers,
```

Comment:
Commented on the other PR, replaying here: headers might not be safe to pass to logging (in case sensitive info is passed), so we might want to exclude it